types/params.go

```diff
@@ -347,6 +350,18 @@ func (params ConsensusParams) ValidateConsensusParams() error {
 	return nil
 }
 
+func (params ConsensusParams) ValidateUpdate(p ConsensusParams, h int64) error {
+	if params.ABCI.VoteExtensionsEnableHeight != 0 && p.ABCI.VoteExtensionsEnableHeight == 0 {
+		return errors.New("vote extensions cannot be disabled once enabled")
+	}
+	if params.ABCI.VoteExtensionsEnableHeight >= h {
```

Comment:
```suggestion
	if params.ABCI.VoteExtensionsEnableHeight >= h || params.ABCI.VoteExtensionsEnableHeight <= h{
```

Reply:
The goal here is to allow chains to update this parameter as long as the height has not already passed. Once the extensions have already started to be created, then we disallow you from disabling them. Do you think this should be done differently?

Reply:
Done, but used the 'updated' field variable for the check.